src/add-card-to-project.ts

```diff
@@ -0,0 +1,67 @@
+import { Probot, Context } from 'probot';
+import { PROJECT_NAME_REGEX, UNSORTED_COLUMN_ALT_NAME_REGEX } from './constants';
+import { LogLevel } from './enums';
+import { log } from './utils/log-util';
+import { EventPayloads } from '@octokit/webhooks';
+
+export function addCardToProject(probot: Probot) {
+  probot.on('issues.labeled', async context => {
+    try {
+      const { payload, octokit } = context;
+      const { name: repo } = payload.repository;
+      const { login: owner } = payload.sender;
+      const { number: issueNumber } = payload.issue;
+      const { name: labelName } = payload.label || {};
+
+      if (labelName == null) {
```

Comment:
We don't need both of these conditionals, and we probably don't want to throw a bespoke error here.

```js
if (!label || !PROJECT_NAME_REGEX.test(label.name)) {
   log('addCardToProject', LogLevel.ERROR, `No project with ${label?.name} exists`);
   return;
}
```

should be sufficient.